packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -30,15 +30,16 @@ export async function openParentExpansionHub(serialNumber: string, moduleAddress
 
     let serial = openSerialMap.get(serialPortPath)!;
 
-    let parentHub = new ExpansionHubInternal(true, serialNumber);
+    let parentHub = new ExpansionHubInternal(true, serial, serialNumber);
```

Comment:
I'm going to be picky and point out that this `serial` variable should also be named `serialPort`.

Reply:
Fixed